### PR TITLE
fix(nav-bar): null check tabs when finding tab

### PR DESCRIPTION
**Pull request by tomaszgrabowski**
## PR Checklist
Please check that your PR fulfills the following requirements:
- [x] The commit message follows [our guidelines](https://github.com/angular/material/blob/master/.github/CONTRIBUTING.md#-commit-message-format)
- [x] Tests for the changes have been added or this is not a bug fix / enhancement
- [x] Docs have been added, updated, or were not required

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Enhancement
[ ] Documentation content changes
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Currently on browser window resize there is a possibility that tabs will not be found on time. This will be ended wit console error `Cannot read property 'length' of null`

Issue Number: 

## What is the new behavior?
No error

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by tomaszgrabowski** [comment]
@googlebot I signed it!

**Comment by tomaszgrabowski** [comment]
@googlebot I fixed it.

**Comment by tomaszgrabowski** [comment]
> Thank you for this issue report and contribution!
> 
> Can you please update the JSDoc and sign the CLA?

@Splaktar required changes uploaded. Thanks for the review.

**Comment by Splaktar** [comment]
CI is broken due to NPM being down atm.

File: src/components/navBar/navBar.spec.js [hunk 1]
```diff
@@ -254,6 +254,17 @@ describe('mdNavBar', function() {
...
+
+    it('does not update selected tab if controller is undefined', function() {
+      $scope.selectedTabRoute = 'tab1';
+
+      spyOn(Object.getPrototypeOf(ctrl), '_updateInkBarStyles');
+      spyOn(Object.getPrototypeOf(ctrl), '_findTab').and.returnValue(null);
+      createTabs();
+
+      expect(ctrl._updateInkBarStyles)
+      .toHaveBeenCalledWith(null, -1, -1);
```
Comment (Splaktar) [inline src/components/navBar/navBar.spec.js:266]: In PR https://github.com/angular/material/pull/11965, I removed the third, unused argument to `_updateInkBarStyles`.
```suggestion
      .toHaveBeenCalledWith(null, -1);
```

**Pull request merged**